src/types.db

```diff
@@ -114,6 +114,19 @@ http_request_methods    value:DERIVE:0:U
 http_requests           value:DERIVE:0:U
 http_response_codes     value:DERIVE:0:U
 humidity                value:GAUGE:0:100
+ib_cap_mask             value:GAUGE:0:U
+ib_errors               value:DERIVE:0:U
+ib_lid                  value:GAUGE:0:U
+ib_lid_mask_count       value:GAUGE:0:U
+ib_link_error_recovery  value:DERIVE:0:U
+ib_octets               rx:DERIVE:0:U tx:DERIVE:0:U
+ib_packets              rx:DERIVE:0:U tx:DERIVE:0:U
+ib_phys_state           value:GAUGE:0:255
+ib_rate                 value:GAUGE:0:U
+ib_sm_lid               value:GAUGE:0:U
+ib_sm_sl                value:GAUGE:0:U
+ib_state                value:GAUGE:0:255
```

Comment:
I'll defer to your expertise on the distinction between `type` and `type_instance`, and am very willing to make this change.

I had consulted [these definitions](https://collectd.org/wiki/index.php/Naming_schema#Plugin_instance_and_type_instance) and felt that these two metrics merited different **types** since each value is a member of a different enum (see [state](https://github.com/torvalds/linux/blob/8c26c87b05323a7ccdc632820b85253e0bf47fd9/drivers/infiniband/core/sysfs.c#L155-L162) vs [phys_state](https://github.com/torvalds/linux/blob/8c26c87b05323a7ccdc632820b85253e0bf47fd9/drivers/infiniband/core/sysfs.c#L294-L303)). Perhaps I'm being too pedantic.

Let me know which you would prefer.

Reply:
Thanks for providing details on state vs. phys_state. Looks like leaving them as different types would be the right thing to do as they indicate complete different set of metrics.